aliyun/ecs/connection.py

```diff
@@ -557,6 +557,8 @@ def create_instance(
             instance_name (str): The name to use for the instance.
             internet_max_bandwidth_in (int): Max bandwidth in.
             internet_max_bandwidth_out (int): Max bandwidth out.
+	    instancechargetype (str): The charge type of the instance, 'PrePaid' or 'PostPaid'.
```

Comment:
It seems this should be instance_charge_type to follow existing naming conventions.